### ISSUE TITLE
IPython/Jupyter notebooks

### ISSUE THREAD

**Issue by femtotrader** [cuemacro/finmarketpy#4]
Hello,

IPython/Jupyter notebooks could be a great tool to use to show some features of pythalesians.

Kind regards

**Comment by saeedamen** [comment]
I don't think it would take me too long to do Jupyter notebooks. In the meantime I'd check out the pythalesians-examples folder, which has lots of scripts to demonstrate the features. I'll try to make Jupyter notebooks a priority (quicker than doing unit tests).

**Comment by saeedamen** [comment]
Just added a notebook. However, I'll work on adding more examples, but hopefully this will help as a start for you!

**Comment by femtotrader** [comment]
Thanks @saeedamen
